<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <widget class="QPushButton" name="import_pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>613</width>
     <height>222</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="ExcelTab">
    <attribute name="title">
     <string>Excel</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="Excel_TableName">
         <property name="placeholderText">
          <string>TableName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExcelBrowse_pushButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="ExcelHeaders_checkBox">
       <property name="text">
        <string>Headers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ExcelWorksheetName_lineEdit">
       <property name="placeholderText">
        <string>Worksheetname</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="CSVTab">
    <attribute name="title">
     <string>CSV</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="CSV_TableName">
         <property name="placeholderText">
          <string>TableName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CSVBrowse_pushButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="CSVHeaders_checkBox">
       <property name="text">
        <string>Headers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Delimeter</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="CSVDelimeterTab_radioButton">
         <property name="text">
          <string>Tab</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="CSVDelimeterSpace_radioButton">
         <property name="text">
          <string>Space</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="CSVDelimeterSemicolumn_radioButton">
         <property name="text">
          <string>;</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="CSVDelimeterComma_radioButton">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="CSVBypass_checkBox">
       <property name="text">
        <string>Bypass to local db table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CSVBypassTablename_lineEdit">
       <property name="placeholderText">
        <string>LocalDB tablename</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
